src/components/Timelog/Timelog.jsx

```diff
@@ -473,8 +476,8 @@ const Timelog = props => {
                       {isOwner ? (
                         <div className="float-right">
                           <div>
-                            <Button color="success" onClick={toggle}>
-                              {'Add Intangible Time Entry '}
+                            <Button color="success" onClick={toggle} style={boxStyle}>
+                              Add Intangible Time Entry{' '}
```

Comment:
```suggestion
                              Add Intangible Time Entry
```
This is probably the caused by the linter.